components/up.libsonnet

```diff
@@ -0,0 +1,79 @@
+local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
+
+{
+  local up = self,
+
+  config:: {
+    name: error 'must provide name',
+    version: error 'must provide version',
+    image: error 'must provide image',
+    backoffLimit: error 'must provide backoffLimit',
+    writeEndpoint: error 'must provide writeEndpoint',
+    readEndpoint: error 'must provide readEndpoint',
+
+    commonLabels:: {
+      'app.kubernetes.io/name': 'observatorium-up',
+      'app.kubernetes.io/instance': up.config.name,
```

Comment:
This is not quite right I think, we end up with both `instance` and `name` being `observatorium-up`.  Instance should not be the name of the application

Reply:
Hmm. That's what we do in a couple of places actually... :thinking: 
https://github.com/thanos-io/kube-thanos/blob/master/jsonnet/kube-thanos/kube-thanos-receive.libsonnet#L17

Reply:
but the difference is that the `name` there ends up being "default" for the default hashringm here it is exactly the same as the app name

Reply:
Isn’t this just coincidental that we happen to name our instance of this observatorium-up component “observatorium-up” ? This seems ok to me.

Reply:
Well it could be coincidental, but then the question is: what does it mean? I don't know what the 'observatorium-up' instance refers to. If we had multiple ups and sorted by instance label, would you know what this one was?

Reply:
If I understand correctly you’re suggesting we should choose a more descriptive name for the instance than “observatorium-up” to reflect its purpose rather than _what_ it is? I’m all for that. Suggestions what to name it?

Reply:
Kind of, the instance label exists to tell us what instance this app is. "Observatorium-up'" is not an instance, it's an app name. "default" is an instance, "testing"could be an instance, "prod" could even be an instance. I guess I'm also not really sure why this is here. Is it only for e2e tests? Then "e2e" can be an instance

Reply:
e2e-test sounds good to me as that is actually it’s purpose here right now. We would have a different name for when we introduce it into the base setup as a Blackbox probe.

Reply:
The label is updated now. 
These definitions will be moved to `observatorium/up` projects eventually. I've put them as they solely needed in here right now.